src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -141,8 +141,17 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
             // Check the item count.
             if (searchParams.Count != null)
             {
+                if (searchParams.Count > _featureConfiguration.MaxItemCountPerSearch)
+                {
+                    throw new BadRequestException($"_count exceeds limit configured for server ({_featureConfiguration.MaxItemCountPerSearch}).");
```

Comment:
Any error messages that could potentially be exposed to a user should be in a Resources file

Reply:
Thank you.
Updated.